RIOT-OS/RIOT
# Pr/add cc26x2 cc13x2 radio

### Contribution description


This PR is a continued work on #13295 from @jeandudey. Basically it retakes the same features and code, I just updated it and adapted it for the current API and HAL. 

### Testing procedure


For now some warnings are being triggered, I'm working on them but I think it will take me a while to remove them, although the code compiles and works with the following command:
```console
$ BOARD=cc1312-launchpad WERROR=0 make -C examples/gnrc_networking flash term
```
I only tested on 2 cc1312-launchpad boards, I'd need to get some other supported sub-ghz board to check broad compatibility.


### Issues/PRs references


#13295 
#13205
#13215
